### PR TITLE
Normalize Event Sourcing wording #29577

### PR DESCRIPTION
**Pull request by jlprat**
References: #29577

Replaces with "Event Sourcing" any occurrence of:
- Event sourcing
- event sourcing
- event Sourcing
- EventSourcing
- Event Source (all case variants included)

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/from-classic.md [hunk 1]
```diff
@@ -388,7 +388,7 @@ Links to reference documentation:
 
 The correspondence of the classic `PersistentActor` is @scala[`akka.persistence.typed.scaladsl.EventSourcedBehavior`]@java[`akka.persistence.typed.javadsl.EventSourcedBehavior`].
 
-The Typed API is much more guided to facilitate event sourcing best practises. It also has tighter integration with
+The Typed API is much more guided to facilitate Event Sourcing best practices. It also has tighter integration with
```
Comment (jlprat) [inline akka-docs/src/main/paradox/typed/from-classic.md:391]: As I was modifying this line, I saw that "practises" uses the English spelling and, AFAIR, Akka uses the American one.

**Comment by johanandren** [comment]
PLS BUILD

**Comment by jlprat** [comment]
The second commit is only to pick Travis attention

File: akka-docs/src/main/paradox/general/message-delivery-reliability.md [hunk 2]
```diff
@@ -202,14 +201,14 @@ actually do apply the best effort to keep our tests stable. A local `tell`
 operation can however fail for the same reasons as a normal method call can on
 the JVM:
 
- * `StackOverflowError`
- * `OutOfMemoryError`
- * other `VirtualMachineError`
+* `StackOverflowError`
+* `OutOfMemoryError`
+* other `VirtualMachineError`
```
Comment (johanandren) [inline akka-docs/src/main/paradox/general/message-delivery-reliability.md:206]: Isn't the space before bullet required (or am I mixing up markup languages again)?

**Review by johanandren — COMMENTED** [review]
Looks good, one question about unrelated change.

File: akka-docs/src/main/paradox/general/message-delivery-reliability.md (hunk 2 above)
Comment (jlprat) [reply akka-docs/src/main/paradox/general/message-delivery-reliability.md:206]: My linter says, no space before a list. Checked locally and it rendered fine

Comment (jlprat) [reply akka-docs/src/main/paradox/general/message-delivery-reliability.md:206]: 0 spaces first level, 2 spaces second level and so on. I think this is how ot woks. But interpreters are very lax on those mistakes.

Like the trailing space, it's 0 or 2.

**Comment by jlprat** [comment]
Failure cased by https://github.com/akka/akka/issues/29858
Unrelated to the change

PLS BUILD

**Review by johanandren — APPROVED** [review]

**Review by raboof — COMMENTED** [review]
Good improvement!

I think we should capitalize the 'r' in 'Replicated Event Sourcing' as well, as it is a term/feature.

**Comment by jlprat** [comment]
@raboof feedback applied! Thanksn

**Review by raboof — APPROVED** [review]

**Pull request merged**